bescort.lic

```diff
@@ -243,7 +249,17 @@ class Bescort
       pause 0.5
     end
   end
-
+	
+	def oshu_manor(mode) 
+		case mode
+		when /exit/i
+      walk_to(2317)
+    when /enter/i
+			wait_for_script_to_complete('oshu_manor')
+			find_room_list(%w(e e w w w n n n))
```

Comment:
oshu_manor.lic's 'empty room' check is entirely to avoid running into higher-circle mobs on the way to the manor. It's basically a long bescort.lic routine that gets the player from one relatively safe room (the **base** room outside of germish'dins ) to just inside the manor (manor's door is locked until the puzzle is solved).